### DISCUSSION TITLE
HDR to HDR transcoding (to reduce bitrate only)

### DISCUSSION THREAD

**Discussion by TheTyrius** [jellyfin/jellyfin#7688] [Q&A]
Hello,

I have HDR capable devices outside the local jellyfin network.
Transcoding is necessary due to bandwidth constraints, but the client supports HEVC/HDR playback.

Is a setup like that feasable at all?
I couldn't find much info on this topic and would appreciate any insights.

Best Regards.

**Comment by Nicolo2524** [comment]
me too i need to transcode something to 1080p and like 15mb of bitrate in hdr, because my tv don't support mkv dolby digital so from transcoding the file to something other than mkv but still dolby digital it works.
